module/Frontpage/view/partial/poll.phtml

```diff
@@ -27,17 +27,31 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
         <?php $text = $this->escapeHtml(($lang === 'en') ? $option->getEnglishText() : $option->getDutchText()); ?>
         <?php $voted = !is_null($userVote) && $option->getId() === $userVote->getPollOption()->getId(); ?>
 
-        <?php if ($showResults || !$canVote): ?>
-            <div>
-                <h5 class="clearfix<?= $voted ? ' text-primary' : '' ?>">
-                    <?= $text ?>
-                    <small class="pull-right" style="margin-top: 0.4em;"><?= $option->getVotesCount() === 1 ? $this->translate('1 vote') : sprintf($this->translate('%d votes'), $option->getVotesCount()) ?></small>
-                </h5>
+        <?php if ($showResults && $canVote): ?>
+            <div class="radio">
+                <label>
+                    <input type="radio" value="<?= $option->getId() ?>" name="option">
```

Comment:
QoL Improvement: centre radio button vertically.

Reply:
This is something for another time I guess